index.js

```diff
@@ -5,7 +5,7 @@ const { join, resolve, sep } = require('path');
 
 parseArgs()
 	.then(state => requireFiles(state))
-	.then(state => spawnNextCommand(state))
+	.then(state => process.exitCode = spawnNextCommand(state).status)
```

Comment:
Could this still 'succeed' (code 0) if the last command is successful even though a prior command failed?